frontend/src/views/GameInvite.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+	<div class="page_box_wrapper">
+		<div class="page_box">
+			<h1>YOU'VE BEEN INVITED</h1>
+			<h1>BY</h1>
+			<h1>PLAYER_02</h1>
```

Comment:
Not something you have to change, but more to know:

in html, a "section" of "content" is supposed to have 1 `<h1>`. Which will be the "main heading" of that section of content. 
This is more a search-engine thing. Since that will take the first h1 to determine the title of the page. So not really relevant for our project.
You could create newlines by using the linebreak element `<br>`

Reply:
And do you have the ID of player 02 at this stage? Than you can show the name, but I'm not sure if the ID is available in this component..